packages/neuron-wallet/electron-builder.yml

```diff
@@ -62,8 +62,11 @@ mac:
   entitlementsInherit: assets/entitlements.plist
   electronLanguages: ["en", "zh", "zh_CN", "zh_TW"]
   target:
-    - dmg
-    - zip
+    - target: default
```

Comment:
Should we retain `dmg` and `zip`? @Keith-CY

Reply:
`target: default` will package a `zip` and a `dmg`

Ref: https://www.electron.build/configuration/mac.html

Reply:
@yanguoyu It seems this value should be a `String`. 
<img width="1407" alt="image" src="https://user-images.githubusercontent.com/22511289/197990809-ab87966b-ec75-4b57-8c9d-50370dfa0c25.png">

Reply:
> @yanguoyu It seems this value should be a `String`. <img alt="image" width="1407" src="https://user-images.githubusercontent.com/22511289/197990809-ab87966b-ec75-4b57-8c9d-50370dfa0c25.png">

Yes, it's a string here

```yml
target:
  - target: default
    arch:
      - arm64
      - x64
```

equals to

```json
{
  target: [{
    target: 'default',
    arch: ['arm64', 'x64']
  }]
}
```

Reply:
I have tried with `default` to package, it will output zip and dmg.